runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/FlinkBatchTransformTranslators.java

```diff
@@ -302,25 +301,8 @@ public void translateNode(Write.Bound<T> transform, FlinkBatchTranslationContext
     }
   }
 
-  private static class GroupByKeyOnlyTranslatorBatch<K, V> implements FlinkBatchPipelineTranslator.BatchTransformTranslator<GroupByKey.GroupByKeyOnly<K, V>> {
```

Comment:
Ah, had a second look. `GroupByKey` has been removed. Should be good to merge as-it-is then.